.idea/codeStyles/Project.xml

```diff
@@ -1,6 +1,22 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```

Comment:
We should not track on GitHub what is in the ./idea folder as this will always change and will appear in every PR of yours. 

Might be a good idea to add this to gitignore.